html/themes/custom/dc_theme/templates/node/node--data-set--full.html.twig

```diff
@@ -194,6 +194,10 @@
       {% if node.field_modified_date and not node.field_modified_date.isempty %}
       <hr>
       <h2>{{ 'Data dictionary'|t }}</h2>
+        {% if node.field_columns and not node.field_columns.isempty %}
+          <p><a href="/node/{{ node.id }}/download/columns/csv">Export data dictionary to csv</a></p>
```

Comment:
Each download link should be a `ul > li`.
Use "CSV" and "XLSX". Better to use the name of the format than the application.
It would be better to generate this list of links in template pre-process and just output the variable here. The variable would just be NULL if there are no columns.
There should be a sngle function the also generates the list for `bc_dc_form_alter()`.